docs/mathematics/prime-number.mdx

```diff
@@ -0,0 +1,124 @@
+---
+title: Prime Number
+slug: /mathematics/prime-number
+---
+
+## Introduction
+
+Prime numbers are the positive integers having only two factors, 1 and the integer itself.
+
+If `p` is a prime, then it’s only factors are necessarily `1` and `p` itself. A number which does not follow this and have more than two factors or which can be factored by other numbers are called composite numbers.
+![How-do-prime-numbers-work](https://user-images.githubusercontent.com/60662775/114220755-05fdfc80-998a-11eb-9a11-5dbdcda3fca5.png)
+
+### Quick Example :
+
+The first ten primes are: `2`, `3`, `5`, `7`, `11`, `13`, `17`, `19`, `23`, `29`.
+
+Here 2 is prime as it's factors are 1 and 2 itself. Similarly 3 and 5 are prime as their factors are 1 and 3 and 1 and 5 respectively.
+`It should be noted that 1 is a non-prime number.`
```

Comment:
No need to keep `It should be noted that 1 is a non-prime number.` in backticks. Just keep `1` in backticks.